reactjs-todo/webpack.config.js

```diff
@@ -94,20 +93,20 @@ module.exports = () => {
       client: {
         overlay: false,
       },
-      port: 8443,
+      port: PORT,
```

Comment:
I don't think this is a problem in this repo - but if someone does put a `.env` file in the _root_ of the workspace, or at least - not in the project specific folder, technically its possible for this `PORT` variable to conflict with the `todo-api` `PORT` variable. I don't think they would get picked up but its something to consider.

We could avoid this by making the `todo-api` read something like `SERVER_PORT` instead to make it clear that these two `PORTS` are different

Reply:
yeah let me change that on the todo-api. I can see this confusing people